### PR TITLE
[CTCP-5576] display gnss longitude and latitude

### PR DESCRIPTION
**Pull request by prashanthmamidi**

### CODE REVIEW

**Comment by prashanthmamidi** [comment]
image

File: app/utils/IncidentP5Helper.scala [hunk 1]
```diff
@@ -62,13 +62,19 @@ class IncidentP5Helper(
...
-  def coordinatesRow: Option[SummaryListRow] = buildRowFromAnswer[String](
-    answer = Some("coordinates"), // TODO: Pull from incident data
-    formatAnswer = formatAsText,
-    prefix = "departure.notification.incident.index.coordinates",
-    id = None,
-    call = None
-  )
+  def coordinatesRow: Option[SummaryListRow] = {
+    val coordinates = data.Location.GNSS.map(
+      gnss => Seq(gnss.latitude, gnss.longitude).mkString(",")
```
[the remark this answers is not in the document]
Comment (jackheslop96) [reply app/utils/IncidentP5Helper.scala:67]: So your `answer` would be `data.Location.GNSS`, and we'd use buildRowFromAnswer[GNSSType] with your newly added format method

[the remark this answers is not in the document]
Comment (prashanthmamidi) [reply app/utils/IncidentP5Helper.scala]: updated

File: app/utils/SummaryListRowHelper.scala [hunk 2]
```diff
@@ -44,6 +44,9 @@ class SummaryListRowHelper(implicit messages: Messages) {
 
   protected def formatAsText[T](answer: T): Content = s"$answer".toText
 
+  protected def formatAsCoordinates(answer: GNSSType): Content =
+    Seq(answer.latitude, answer.longitude).mkString(",").toText
```
Comment (jackheslop96) [inline app/utils/SummaryListRowHelper.scala:48]: @prashanthmamidi do you know where this requirement came from? Or is it just a guess? I can't see anything on the ticket. In the IE015 we formatted it like [this](https://github.com/hmrc/ctc-departure-route-details-frontend/blob/44b5baddc9979c765f9b7d98ed483924a7442c58/app/models/Coordinates.scala#L25).

Might be worth checking with Gavin and Nige

Comment (prashanthmamidi) [reply app/utils/SummaryListRowHelper.scala:48]: @jackheslop96  it's just a guess - I think it matches with what we done in IE015 - latitude, longitude.
Will check with Gavin and see if there's any specific format we need to display

Comment (prashanthmamidi) [reply app/utils/SummaryListRowHelper.scala]: Updated as (latitude, longitude) format

**Comment by prashanthmamidi** [comment]
image

**Review by jackheslop96 — DISMISSED** [review]

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**